app/assets/stylesheets/navbar.css

```diff
@@ -9,6 +9,9 @@ nav i, nav [class^="mdi-"], nav [class*="mdi-"], nav i.material-icons {
   height: 50px;
 }
 
+.navbar-menu li>a{
```

Comment:
@kushalnl7 could you remove this style? I believe it is unnecessary after the changes to `metrics.scss`